tools/clang/tools/dotnetc/EditorForm.cs

```diff
@@ -624,7 +624,10 @@ private void CompileDocument()
                 {
                     try
                     {
-                        var result = compiler.Compile(source, fileName, fileVars.Entry, fileVars.Target, new string[] { "-ast-dump" }, 1, null, 0, library.CreateIncludeHandler());
+                        List<string> args = new List<string>();
+                        args.Add("-ast-dump");
+                        args.AddRange(tbOptions.Text.Split());
+                        var result = compiler.Compile(source, fileName, fileVars.Entry, fileVars.Target, args.ToArray(), args.Count, null, 0, library.CreateIncludeHandler());
```

Comment:
nit: line wrapping? This seems like an excessively long line. I know it is C# not C++, but having more consistent formatting would be nice.

Reply:
Did it manually.
Ctrl + K, Ctrl + F not work :(